bitnami/fluentd/README.md

```diff
@@ -0,0 +1,140 @@
+| `metrics.service.port`                          | Prometheus metrics service port                                                                                | `24231`                                                 |
+| `metrics.service.annotations`                   | Annotations for Prometheus metrics service                                                                     | `{}`                                                    |
+| `tls.enabled`                                   | Enable the addition of TLS certificates                                                                        |                                                         | `false`     |
+| `tls.caCertificate`                             | Ca certificate                                                                                                 | Certificate Authority (CA) bundle content               |
+| `tls.serverCertificate`                         | Server certificate                                                                                             | Server certificate content                              |
+| `tls.serverKey`                                 | Server Key                                                                                                     | Server private key content                              |
+| `tls.existingSecret`                            | Existing secret with certificate content                                                                       | `nil`                                                   |
+| `securityContext.enabled`                       | Enable security context                                                                                        | `true`                                                  |
+| `securityContext.fsGroup`                       | Group ID for the container filesystem                                                                          | `1001`                                                  |
+| `securityContext.runAsUser`                     | User ID for the container                                                                                      | `1001`                                                  |
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+
+```console
+$ helm install --name my-release \
+  --set aggregator.port=24444 bitnami/fluentd
+```
+
+The above command sets the aggregators to listen on port 24444.
+
+Alternatively, a YAML file that specifies the values for the parameters can be provided while installing the chart. For example,
+
+```console
+$ helm install --name my-release -f values.yaml bitnami/fluentd
+```
+
+> **Tip**: You can use the default [values.yaml](values.yaml)
+
```

Comment:
Can you scale this solution without using helm?

Reply:
What do you mean?

Reply:
Can you scale the solution running `kubectl scale sts --replicas Y`? For instance, if **daemonset's pods** won't forward traffic to the new replicas because the "forwarder-configmap" is not recalculated unless you use `helm upgrade xxxx bitnami/fluentd --set replicaCount=Y`, then we need to document it.
See https://github.com/bitnami/charts/tree/master/bitnami/etcd#production-and-horizontal-scaling

Reply:
Ok, now I get it. With the default configuration, `kubectl scale` won't work for the reasons you said. With a custom configuration, it may work. I've added a section to the README.